app/mailers/proposals_mailer.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class ProposalsMailer < ApplicationMailer
+  include Decidim::SanitizeHelper
+  include Decidim::ComponentPathHelper
+
+  layout "decidim/mailer"
+
+  def notify_massive_import(proposals_collection, user)
+    @organization = user.organization
+    @participatory_space_title = decidim_sanitize_translated(proposals_collection.first.participatory_space.title) if proposals_collection.first.present?
+    @user = user
+    @component_url = manage_component_url(proposals_collection.first.component)
+    I18n.locale = user.locale if user.locale.present?
+
+    subject = I18n.t("proposals_publisheds_for_space.email_subject", scope: "decidim.events.proposals", participatory_space_title: @participatory_space_title)
```

Comment:
what is publisheds? 🤔 I think is incorrect..

Reply:
Done like previous suggestion, thanks!